build-tools/installers/create-installers.targets

```diff
@@ -119,6 +148,7 @@
     <_FrameworkFilesWin Include="$(FrameworkSrcDir)\$(BclFrameworkVersion)\Xamarin.Android.NUnitLite.xml" />
   </ItemGroup>
   <ItemGroup>
+    <_MSBuildFiles Include="@(JIUtilityFile->'$(MSBuildSrcDir)\%(Identity)')" />
```

Comment:
I think we'll want to set the `ExcludeFromAndroidNETSdk="true"` metadata on this entry to make sure it won't be included in the Microsoft.Android.Sdk packs.